lib/system/iterators_1.nim

```diff
@@ -178,7 +180,7 @@ else: # not defined(nimNewRoof)
       inc i
 
 
-iterator `||`*[S, T](a: S, b: T, annotation: static string = "parallel for"): T {.
+iterator `||`*[S, T](a: S, b: T, annotation: static string = "omp parallel for"): T {.
```

Comment:
If you need a new iterator type, make one, don't abuse the parallel for iterator.

Removing the `omp` as a prefix of the iterator annotation is a breaking change for anyone using this iterator with a custom annotation.

Reply:
I dont understand what you mean, the `||` keeps working.

Reply:
Now anyone that use `||` with a custom annotation must prefix their annotation with `omp`, so this is a breaking change.

@mratsim's laser is [a big user of this iterator](https://github.com/numforge/laser/blob/master/laser/openmp.nim), and this change will cause it not to compile.

Reply:
My advise would be to create a brand new loop type. And note that `#pragma unroll` can unroll any loop, so a solution that allows tagging a generated loop would be much more efficient. Basically implementing the `{.unroll.}` pragma, if you may.